lib/src/command/lish.dart

```diff
@@ -381,3 +477,26 @@ String _readableFileSize(int size) {
     return '<1 KB';
   }
 }
+
+class _Publication {
+  Uint8List packageBytes;
+  int warningsCount;
+  int hintsCount;
```

Comment:
nit: I think it is more right, but not 100% sure... not important :D
```suggestion
  int warningCount;
  int hintCount;
```